src/lib/viewers/text/TextBaseViewer.js

```diff
@@ -55,17 +62,23 @@ class TextBaseViewer extends BaseViewer {
         let newFontSize = 0;
 
         if (inOrOut === 'in') {
-            newFontSize = `${size + 10}%`;
+            newFontSize = size + 10;
         } else if (inOrOut === 'out') {
-            newFontSize = `${size - 10}%`;
+            newFontSize = size - 10;
         }
 
-        el.style.fontSize = newFontSize;
+        el.style.fontSize = `${newFontSize}%`;
```

Comment:
Can this lead to a value of `0%` due to the default value of `newFontSize`? I would probably revert this change, since the duplication of the template isn't super impactful.

Reply:
Yeah `0%` is possible, but that is no different from the behavior it was before. This change was only to keep `newFontSize` as a number which could be passed into the `zoomControls.setCurrentScale` call on L76